### ISSUE TITLE
Failure to parse query when dot follows semicolon

### ISSUE THREAD

**Issue by tpluscode** [oxigraph/oxigraph#943]
**Describe the bug**

Oxigraph fails to parse valid syntax

**To Reproduce**
1. Node `oxigraph` v0.4.0-alpha.7

```js
import oxigraph from 'oxigraph'

(async function() {
  const store = new oxigraph.Store()
  store.query(`PREFIX schema: <http://schema.org/>
  
  CONSTRUCT WHERE {
    ?s 
      schema:name ?name ;
    .
  }`)
})()
```

**Comment by Tpt** [comment]
This is indeed a bug. It is that this case is not covered by the official testsuite. I am going to fix it soon.

**Comment by tpluscode** [comment]
Thank for acting quick. While the issue is still open, I'd add that just now noticed a similar problem when an anonymous blank node has a semicolon at the last prop in CONSTRUCT

```sparql
import oxigraph from 'oxigraph'

(async function() {
  const store = new oxigraph.Store()
  store.query(`PREFIX schema: <http://schema.org/>
  
CONSTRUCT {
 [ 
    schema:name ?name ;
 ] .
 } WHERE {
    [ 
      schema:name ?name ;
    ] .
  }`)
})()
```

**Comment by Tpt** [comment]
The already merged fix seems to also fix this error

**Comment by Tpt** [comment]
Released as part of 0.4.0-alpha.8
